index.html

```diff
@@ -5,6 +5,6 @@
     <title>jamaica</title>
   </head>
   <body>
-
+<h1>first challenge</h1>
```

Comment:
i will comment here in future to show you how things can be done better etc

Reply:
ok thanks